frontend/src/views/web/challenge/challenge-page.html

```diff
@@ -48,14 +48,22 @@
             <div class="col m3 l3 align-right">
                 <button class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14" type="submit" ng-click="challenge.starChallenge()" ng-init="challenge.isStarred()"><i class="fa fa-star star"></i> {{challenge.data}}&nbsp;&nbsp;{{challenge.count}}</button>
             </div>
+            <div ng-if="challenge.isChallengeHost" style="padding-top: 30px;" class="col m3 l3 align-right">
+                <div style="display: inline-block;" ng-click="challenge.toggleParticipation($event, challenge.canParticipate)"
+                    class="btn-switch" ng-class="{'btn-switch--on':challenge.canParticipate}" ng-model="challenge.canParticipate">
```

Comment:
I think you missed updating this - `challenge.canParticipate`

Reply:
Please update it everywhere.

Reply:
yeah I noticed that after pushing :)

Reply:
@RishabhJain2018 done